src/contrib/cluster/Akka.DistributedData.LightningDB/LmdbDurableStore.cs

```diff
@@ -207,12 +192,13 @@ private void Init()
                 }
 
                 var t0 = Stopwatch.StartNew();
-                using (var env = GetLightningEnvironment())
-                using (var tx = env.BeginTransaction(TransactionBeginFlags.ReadOnly))
-                using(var db = tx.OpenDatabase(DatabaseName))
-                using(var cursor = tx.CreateCursor(db))
+                
+                try
                 {
-                    try
+                    using (var env = GetLightningEnvironment())
```

Comment:
Might as well re-arrange the try...catch block to include the LMDB environment initialization code so it can catch exceptions thrown there